<p>A class should not have so many methods</p>
<h2>Noncompliant code example</h2>
<pre>
function functionUnderTest1(){
a=1
}
function functionUnderTest2(){
a=2
}
function functionUnderTest3(){
a=3
}
function functionUnderTest4(){
a=4
}
function functionUnderTest5(){
a=5
}
function functionUnderTest6(){
a=6
}
function functionUnderTest7(){
a=7
}
function functionUnderTest8(){
a=8
}
function functionUnderTest9(){
a=9
}
function functionUnderTest10(){
a=10
}
function functionUnderTest11(){
a=11
}
function functionUnderTest12(){
a=12
}
function functionUnderTest13(){
a=13
}
function functionUnderTest14(){
a=14
}
function functionUnderTest15(){
a=15
}
</pre>
