<p>Your code should not have too long lines. Break them into shorter lines</p>
<h2>Noncompliant code example</h2>
<pre>
        double calculation = strinbB.length()*strinbB.length()* Math.random()*10 + stringA.length()*Math.random()*10 + strinbB.length()*Math.random()*10 + Math.random()*10 * Math.max(stringA.length(),strinbB.length());// Noncompliant {{Your code should not have too long lines. Break them into shorter lines}}
</pre>
