<p>A Method should not have so many conditions</p>
<h2>Noncompliant code example</h2>
<pre>
     public void methodTestIncorrect()//Noncomplaint {{A Method should not have so many conditions}}
    {
        int testValueA = 1;
        int testValueB = 4;
        if(testValueA==0)
        {

        }
        switch(testValueA)
        {
            case 1: ;
            case 2: ;
            case 3: ;
        }

        if(testValueB ==4)
        {

        }
    }
</pre>
